### PR TITLE
Define defaultStateScriptTimeout const, to be consistent with the rest

### PR DESCRIPTION
**Pull request by lluiscampos**
Changelog: None

Signed-off-by: Lluis Campos <lluis.campos@northern.tech>

### CODE REVIEW

File: statescript/executor.go [hunk 1]
```diff
@@ -72,6 +74,16 @@ func (l *Launcher) getRetryTimeout() time.Duration {
 	return defaultStateScriptRetryInterval
 }
 
+func (l Launcher) getTimeout() time.Duration {
+
+	if l.Timeout != 0 {
+		return time.Duration(l.Timeout) * time.Second
+	}
+	log.Debug("statescript: timeout for executing scripts is not defined; " +
```
Comment (kacf) [inline statescript/executor.go]: Need `Debugf` here.

Comment (lluiscampos) [reply statescript/executor.go]: Thanks. Travis agrees.

**Comment by lluiscampos** [comment]
I added the change of the default value as criteria for [MEN-2409](https://tracker.mender.io/browse/MEN-2409).

**Comment by lluiscampos** [comment]
Created a unit test to **not decrease** coverage. I will merge tomorrow after the integration tests results

**Pull request merged**